### PR TITLE
BD-17 feat: sign-in markup

### PR DESCRIPTION
**Pull request by sfgardev**

### CODE REVIEW

File: pages/sign-in/index.tsx [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
 
 const SignIn = () => {
-  return <div className='text-center text-white'>SignIn</div>
+  return (
+    <section className='grid place-items-center px-4 pt-9 text-light-100'>
+      <Card className='w-full max-w-[378px]'>
+        <header className='mb-6 flex flex-col gap-3'>
+          <Typography.H1 className='text-center'>Sign In</Typography.H1>
+          <div className='flex justify-center gap-16'>
+            <Icon height={36} icon='google' width={36} />
```
Comment (SlavaMryasov) [inline pages/sign-in/index.tsx:13]: add links

File: pages/sign-in/index.tsx [hunk 2]
```diff
@@ -1,7 +1,42 @@
...
+            <Icon height={36} icon='github' width={36} />
+          </div>
+        </header>
+        <form className='mb-[18px]'>
+          <div className='mb-9 flex flex-col gap-6'>
+            <Input label='Email' name='email' placeholder='Example@gram.com' type='email' />
+            <PasswordInput label='Password' name='password' placeholder='************' />
```
Comment (SlavaMryasov) [inline pages/sign-in/index.tsx:20]: change placeholder text

**Review by SlavaMryasov — CHANGES_REQUESTED** [review]
fix placeholder and add links

File: pages/sign-in/index.tsx (hunk 1 above)
Comment (sfgardev) [reply pages/sign-in/index.tsx:13]: it is not final variant, these icons will be needed for oauth (github and google reg flow)

File: pages/sign-in/index.tsx (hunk 2 above)
Comment (sfgardev) [reply pages/sign-in/index.tsx:20]: what text?

**Review by SlavaMryasov — APPROVED** [review]

**Pull request merged**